packages/core/src/mixins/source.js

```diff
@@ -112,8 +112,9 @@ module.exports = mixin(
                 const sourcePath = this.fullPath;
                 if (!this._monitor && sourcePath) {
                     Log.debug(`Watching ${this.name} directory - ${sourcePath}`);
+                    const exclude = this.get('exclude') || [];
```

Comment:
`get` has a second argument for the default value but somehow this isn’t used much anywhere…

Reply:
As `get` is not typed, specifying the default value outside with or operator gives code editors a bit more knowledge about the variable, I think.